src/test/java/org/candlepin/model/PoolCuratorTest.java

```diff
@@ -1700,13 +1702,13 @@ public void testGetPoolsOrderedByProductNameAscending() {
         Owner owner1 = this.createOwner();
 
         Pool p1 = TestUtil.createPool(owner1, this.generateProduct(owner1, "p1", "xyz"))
-            .setSourceSubscription(new SourceSubscription("subscriptionId-phil", "master"));
+            .setSourceSubscription(new SourceSubscription("subscriptionId-phil", "primary"));
```

Comment:
value change (several others throughout this file; omitting additional comments)

Reply:
done